### PR TITLE
ADD | NIMB BEP20 token

### PR DESCRIPTION
**Pull request by Sam-OConnor**

### CODE REVIEW

**Comment by Cryptocool1** [comment]
As per our policy, there is a minimum requirement of 10,000 holder addresses and 15,000 transactions, in order to filter out scam tokens.
See the criteria at : https://developer.trustwallet.com/listing-new-assets/requirements .

**Comment by Sam-OConnor** [comment]
@Cryptocool1 Hello. May I ask if it is possible to add token as is? We are the same company, who added NBU and GNBU, so you can see, it's not some one day scam situation here.

[NBU ERC-20 PR](https://github.com/trustwallet/assets/pull/17219)
[GNBU BEP-20 PR](https://github.com/trustwallet/assets/pull/17218)

Best Regards.

**Review by Lere001 — COMMENTED** [review]
All good

File: blockchains/smartchain/assets/0xCb492C701F7fe71bC9C4B703b84B0Da933fF26bB/info.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          "name": "youtube",
+          "url": "https://youtube.com/channel/UCOl3hXqESQ54chKSn5nsfPg"
+      }
+  ],
+  "tags": [
+      "governance",
+      "staking",
+      "defi"
+  ]
+}
```
Comment (aminpsycho) [inline blockchains/smartchain/assets/0xCb492C701F7fe71bC9C4B703b84B0Da933fF26bB/info.json:30]: tnx

**Pull request closed without merging**